.kitchen.yml

```diff
@@ -75,7 +75,8 @@ platforms:
       image: debian:8
       privileged: true
       provision_command:
-        - echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
+        - echo "deb http://archive.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
+        - echo 'Acquire::Check-Valid-Until "false";' > /etc/apt/apt.con
```

Comment:
Should this be `/etc/apt/apt.conf`?

Reply:
fixed in https://github.com/elastic/ansible-elasticsearch/pull/556/commits/7488a569ac9925778f875e803ea8e96e36982d2b